client/src/App.js

```diff
@@ -1,27 +1,67 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import "./App.css";
 import { BrowserRouter, Switch, Route } from "react-router-dom";
 import Reusable from "./pages/Reusable";
 import Favorites from "./pages/Favorites";
 import Homemade from "./pages/Homemade";
 import Profil from "./pages/Profil";
 import Navigation from "./components/Navigation";
+import saveToLocal from "./lib/saveToLocal";
+import loadFromLocal from "./lib/loadFromLocal";
```

Comment:
```suggestion
import React, { useState, useEffect } from "react";
import { BrowserRouter, Switch, Route } from "react-router-dom";
import Reusable from "./pages/Reusable";
import Favorites from "./pages/Favorites";
import Homemade from "./pages/Homemade";
import Profil from "./pages/Profil";
import Navigation from "./components/Navigation";
import saveToLocal from "./lib/saveToLocal";
import loadFromLocal from "./lib/loadFromLocal";
import "./App.css";
```

Imports should be structured as follows:
1. External libraries 
2. Your own pages / components
3. CSS